changelog/20864.txt

```diff
@@ -0,0 +1,5 @@
+```release-note:bug
+secrets/transit: Fix export of HMAC-only key, correctly exporting the key used for sign operations. For consumers of the previously incorrect key, use the plaintext export to retrieve these incorrect keys and import them as new versions.
```

Comment:
FYI - these didn't format properly in the changelog. I think you might need a different backtick section for each bullet. Can you check the docs? Don't fix this file, just for next time.

Reply:
Interesting, thank you! I thought I had seen an example like this, but maybe it was also fixed; perhaps we should fix this one, so that we know its not correct?

https://github.com/hashicorp/vault/blob/main/changelog/README.md doesn't really give any indication of how to format changelog files with multiple impacts, probably worth calling out, tbh.

Reply:
Ah, the upstream docs do call it out: https://github.com/hashicorp/go-changelog#change-file-formatting : 

> Sometimes PRs have multiple changelog entries associated with them. In this case, use multiple blocks.